python/image-content-search/src/imageAnalysis/main.py

```diff
@@ -1,14 +1,14 @@
 import boto3
 import botocore.config
-import os, sys
+import os
 import logging
 import json
 
 aws_config = botocore.config.Config(
     region_name = os.getenv('REGION'),
     signature_version = 'v4',
     retries = {
-        'max_attempts': int(os.getenv('DEFAULT_MAX_CALL_ATTEMPTS')),
+        'max_attempts': int(os.getenv('DEFAULT_MAX_CALL_ATTEMPTS', '1')),
```

Comment:
If this is empty, but set, this will fail. Python considers an empty environment variable to be set and have a value:

```
$ BANANA= python3
Python 3.8.1 (v3.8.1:1b293b6006, Dec 18 2019, 14:08:53)
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> int(os.getenv('BANANA', 1))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: ''
```

Consider `os.getenv('DEFAULT_MAX_CALL_ATTEMPTS') or '1'`